### PR TITLE
Simplified Readme and Markdown syntax check

### PR DESCRIPTION
**Pull request by dkastl**
I cleaned up the Readme file and removed outdated (and wrong) content. 

The first paragraphs had been aligned, which requires to use HTML content in Markdown, which I would like to avoid, if possible. I prefer to use default styling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,10 +55,12 @@ $ npm install
 $ ELECTRON_MIRROR="https://npm.taobao.org/mirrors/electron/" npm install
 ```
 
-## License
-MIT
+## Licenses
+
+[MIT License](https://github.com/dat-land/dat-desktop/blob/master/LICENSE)
```
Comment (martinheidegger) [inline README.md]: Afaik many markdown implementations now support `[./LICENSE]` which would prevent us from having to rewrite it whenever that might change.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,52 @@
...
-$ npm install             # install dependencies
-$ npm start               # start the application
-$ npm run dist            # compile the app into an ASAR file to release
-$ npm start --data=<dir>  # change the path for new dat archives (default ~/Downloads/dat)
-$ npm start --db=<dir>    # change the path for users settings (default ~/.dat-desktop)
+To run _Dat Desktop_ in development mode:
+
+```sh
+npm install             # install dependencies
+npm run start           # start the application
```
Comment (martinheidegger) [inline README.md]: `npm` has the short-cut `start` → `npm start` should work.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,52 @@
...
 - [License](#license)
 
 ## Download
 
-To run the app locally check out our [download
-guide](https://datproject.org/install) or install directly from the command
-line using [Homebrew Cask](https://caskroom.github.io). Currently only MacOS is
-supported.
+_Dat_ needs to be installed first. To run the app locally check out our [download
+guide](https://datproject.org/install) or install directly from the command line:
```
Comment (martinheidegger) [inline README.md]: I never tried installing it through homebrew-cask. Do we have to do something to update them? Should we create issues to deploy DAT for debian/windows-store/mac-store/etc. ?

File: README.md [hunk 4]
```diff
@@ -1,58 +1,52 @@
...
-    <img src="https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square"
-      alt="Standard" />
-  </a>
-</div>
+# Dat Desktop
+
+**Peer to peer data sharing app built for humans.**
+
+[!["Build Status"](https://img.shields.io/travis/dat-land/dat-desktop/master.svg?style=flat-square)](https://travis-ci.org/dat-land/dat-desktop) 
+[!["Standard](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square)](https://standardjs.com)
```
Comment (martinheidegger) [inline README.md]: I am sure there were aesthetic reasons for this to be there. I am kind-of okay with this change but am not sure how design-people feel about it.

File: README.md [hunk 5]
```diff
@@ -1,58 +1,52 @@
...
 - [FAQ](#faq)
 - [License](#license)
 
 ## Download
 
-To run the app locally check out our [download
-guide](https://datproject.org/install) or install directly from the command
-line using [Homebrew Cask](https://caskroom.github.io). Currently only MacOS is
-supported.
+_Dat_ needs to be installed first. To run the app locally check out our [download
```
Comment (martinheidegger) [inline README.md]: I believe the DAT homepage references https://github.com/dat-land/dat-desktop/releases when it points out how to download DAT. shouldn't we do same?

File: README.md (hunk 4 above)
Comment (dkastl) [reply README.md]: If someone wants to change this, let them change it. In Visual Code the Markdown syntax check underlines plain HTML, and I don't like underline code ;-)

IMO this is to small thing to spend much time on.

File: README.md (hunk 5 above)
Comment (dkastl) [reply README.md]: Any additional content is fine. I just wanted to remove, what was either outdated or didn't look necessary.

File: README.md (hunk 4 above)
Comment (martinheidegger) [reply README.md]: Alright, let's leave it at that

**Pull request merged**